frontend/packages/client/src/components/CommunityProposals.js

```diff
@@ -5,11 +5,14 @@ import { useCommunityProposalsWithVotes, useMediaQuery } from "../hooks";
 import { FilterValues } from "../const";
 import DropDownFilter from "./ProposalsList/DropdownFilter";
 import WrapperResponsive from "./WrapperResponsive";
+import { omit } from "lodash";
```

Comment:
To prevent the entire lodash library from being loaded you can directly import methods to improve tree-shaking during the build process.

`import omit from 'lodash/omit'`

Reply:
removed lodash and using filter